SPECS/curl/curl.spec

```
@@ -89,6 +90,9 @@ rm -rf %{buildroot}/*
 %{_libdir}/libcurl.so.*
 
 %changelog
+*   Thu Jun 24 2021 Suresh Babu Chalamalasetty <schalam@microsoft.com> - 7.76.0-4
+-   CVE-2021-22897 fix
```

Should we be consistent in the spaces :)
```suggestion
* Thu Jun 24 2021 Suresh Babu Chalamalasetty <schalam@microsoft.com> - 7.76.0-4
- CVE-2021-22897 fix
```